emf-builtins/plugin.xml

```diff
@@ -10,12 +10,5 @@
       </package>
    </extension>
 
-   <extension point="org.eclipse.emf.ecore.generated_package">
-      <!-- @generated builtins -->
-      <package
-            uri="http://lionweb.io/lionweb-java/emf/core/builtins/2023.1"
-            class="io.lionweb.emf.builtins.BuiltinsPackage"
-            genModel="src/main/model/builtins.genmodel"/>
-   </extension>
-
+null
```

Comment:
I'm guessing this is a change that's unrelated to the rest of the PR. What's its effect?